### PR TITLE
Incorporate PR text into the model

### PR DESCRIPTION
**Pull request by gewoonrik**
I have written a multi input and output NN using the Keras Functional API [[1]](https://keras.io/getting-started/functional-api-guide/#multi-input-and-multi-output-models)

There are two inputs:

1. diff input
2. comment input

each of those inputs is fed to an LSTM. The output vectors of those LSTM's are merged and fed through multiple dense layers (hardcoded with 64 neurons each, we should experiment with these).

The network has three outputs, one main output and one auxiliary outputs for each LSTM.
These auxiliary outputs are used to also train the LSTM's directly instead of only relying on gradients  from the main output.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -59,9 +83,9 @@
     checkpoint = ModelCheckpoint(checkpoint_file % args.prefix, monitor='val_loss')
     callbacks.insert(checkpoint)
 
-model.fit(x_train, y_train, batch_size=args.batch_size, nb_epoch=args.epochs,
-          validation_data=(x_val, y_val), callbacks=callbacks)
+model.fit([diff_train, comment_train], y_train, batch_size=args.batch_size, nb_epoch=args.epochs,
+          validation_data=([diff_val, comment_val], y_val), callbacks=callbacks)
 
-score, acc = model.evaluate(x_val, y_val, batch_size=args.batch_size)
+score, acc = model.evaluate([diff_val, comment_val], y_val, batch_size=args.batch_size)
```
Comment (gewoonrik) [inline train.py]: I think model.evaluate will return an array instead of scalars, to work with the multiple outputs, but I'm not sure about that yet.

**Comment by gewoonrik** [comment]
Preprocessing for comments still has to be implemented
